src/FilterMonitor.js

```diff
@@ -10,10 +10,11 @@ export default class FilterMonitor extends Component {
     blacklist: PropTypes.array
   };
 
-  isFiltered(action) {
+  isFiltered(actionType) {
+    var type = actionType || '';
```

Comment:
I think so. I'm not sure as I didn't write the original code but to me it looks like the isFiltered method is checking whether there is a match on the whitellist or not one on the blacklist and your return !actionType doesnt tell you if that's the case. You may as well not have the method if you don't match. Can accept the PR or rewrite as you see fit :)